### ISSUE TITLE
TTS Tech Portfolio - How We Work: Tooling

### ISSUE THREAD

**Issue by its-a-lisa-at-work** [18F/tts-tech-portfolio#493]
## Background information

I picked up Project Management tasks and have been trying to do reporting to present at reviews and to make sprint planning and grooming easier -- the current tooling we have feels limited. 

There was a conversation in the #Engineering-Solutions about project management tools that sent me down a rabbit hole. Multiple Project Management tools came up -- ZenHub, Azure Devops, JIRA, GitHub Projects, etc -- so I started to investigate the utility of the tools and what would fit for our TTS Tech Portfolio team. I came across JIRA service desk and considering that we do that type of work, requested access to what GSA has to understand if that is a part of the suite of tools we have available to us. 

## User stories

As a Project Manager for TTS Tech Portfolio, I would like to be able to generate reports based on our issues so that I can track progress

As a Project Manager for the TTS Tech Portfolio, I would like to be able to use a tool that enables priority. 

## Related Issues
People:
Process: 
Tools: https://github.com/18F/tts-tech-portfolio/issues/469

## Acceptance criteria

-Understand how the tooling will help enable the people who are doing the process

---

The assignee should add some checkboxes as a "sketch" of the steps to complete, which may evolve.

**Comment by afeld** [comment]
What's the deliverable for this one? A writeup?

I think it would help to start off with some Requirements Gathering 😄 Some example questions:

- What pain points / limitations do we have with the current tooling?
- What questions do we wish were easier to answer?
- What do we like about our current tooling, and would miss if we switch?

**Comment by its-a-lisa-at-work** [comment]
`[Conversation](https://gsa-tts.slack.com/archives/C03JN1M6S/p1592501775051300)` to contribute to this story

**Comment by its-a-lisa-at-work** [comment]
This is blocked by Zenhub

**Comment by its-a-lisa-at-work** [comment]
User Story:

As someone who is working in the TTS Tech Portfolio - I would like better tooling to chain issues together so that I can better keep track of what needs to be done. 
 -  Currently, I am doing tasks that are discreet and the acceptance criteria is pretty basic; but the tasks feed into a larger effort. As it stands, I don't have any way of keeping track of what is what. This prevents me from working on things at the smallest level.

**Comment by JJediny** [comment]
Closing in favor of https://github.com/18F/tts-tech-portfolio/issues/1237 we will be better able to identify tools as direct links etc then just picking a toolbox to point to
